streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStoreTest.java

```diff
@@ -101,6 +103,53 @@ private void init() {
         metered.init(context, metered);
     }
 
+    @SuppressWarnings("unchecked")
+    @Test
+    public void shouldGetSerdesFromConfigIfNoUserSerdes() {
```

Comment:
A verb is normally expected following 'if no xx'
How about naming this test:

shouldGetSerdesFromConfigWithouUserSerdes